charts/vald/templates/index/job/correction/cronjob.yaml

```diff
@@ -39,7 +38,21 @@ spec:
         metadata:
           labels:
             app: {{ $corrector.name }}
+          annotations:
+            {{- $pprof := default .Values.defaults.server_config.metrics.pprof $corrector.server_config.metrics.pprof -}}
+            {{- if $pprof.enabled }}
+            pyroscope.io/scrape: "true"
+            pyroscope.io/application-name: {{ $corrector.name }}
+            pyroscope.io/profile-cpu-enabled: "true"
+            pyroscope.io/profile-mem-enabled: "true"
+            pyroscope.io/port: {{ $pprof.port | quote }}
+            {{- end }}
         spec:
+          {{- if $corrector.initContainers }}
+          initContainers:
```

Comment:
> **NOTE**
Depending on the scheduling conditions of cron, processing may start before the agent is launched. Therefore, it is necessary to wait for it to start up.